test.yaml

```diff
@@ -0,0 +1,113 @@
+version: '2.3'
+
```

Comment:
can you toss a comment in here about the mesocast structure and what this yaml is doing?

Reply:
I docker compose mesocast but the nexrad 3 and nexrad 2 parsers are stateful and I wanted to test Cedana c/r of those containers specifically, since mesocast is being moved to EKS, I thought it be perfect for testing.

Reply:
The mesocast parsers are like extreme for k8s testing though, which is good, they need minimum 12 cpus and 96 gb of ram, they on average use up 60gb